src/sentry/utils/concurrent.py

```diff
@@ -57,19 +57,24 @@ def worker():
 
         self.__started = True
 
-    def submit(self, callable, *args, **kwargs):
+    def submit(self, callable, priority=0, block=True, timeout=None):
```

Comment:
This follows the defaults from `Queue.put`: https://docs.python.org/2/library/queue.html#Queue.Queue.put